.travis.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - stage: codecoverage
         if: type IN (pull_request) OR branch in (master, develop, hotfix)
         php: 7.0
-        script: ./vendor/bin/codecept run install --env travis-ci-hub -f -vvv -d ; ./vendor/bin/robo -vvv code:coverage; cat codeception.yml; bash <(curl -s https://codecov.io/bash) -f ./_output/coverage.xml
+        script: ./vendor/bin/codecept run install --env travis-ci-hub -f -d --ext DotReporter; echo "\$sugar_config['state_checker']['test_state_check_mode'] = 1;" >> config_override.php  ; ./vendor/bin/robo -vvv code:coverage; cat codeception.yml; bash <(curl -s https://codecov.io/bash) -f ./_output/coverage.xml
```

Comment:
state_checker should be removed: https://github.com/salesagility/SuiteCRM/commit/c9389907d211e7d9f295d930e7bea6bfe17decec. The ```-d``` flag is also unncessary but that can be fixed by #7100